packages/hoppscotch-common/src/components/collections/ImportExport.vue

```diff
@@ -435,26 +435,15 @@ const HoppTeamCollectionsExporter: ImporterOrExporter = {
       )
 
       if (E.isRight(res)) {
-        const { exportCollectionsToJSON } = res.right
-
-        if (!JSON.parse(exportCollectionsToJSON).length) {
-          isHoppTeamCollectionExporterInProgress.value = false
-
-          return toast.error(t("error.no_collections_to_export"))
-        }
```

Comment:
The case where there are no collections to export is now handled via the `left` case, and in the `else` block below an error toast is shown with `res.left` straightaway.